scripts/spack/packages/tribol/package.py

```diff
@@ -30,26 +29,26 @@ def get_spec_path(spec, package_name, path_replacements={}, use_bin=False):
 
 # NOTE: Cannot inherit from builtin until tribol becomes a builtin spack package within spack itself.
 
-class Tribol(CachedCMakePackage, CudaPackage):
+class Tribol(CachedCMakePackage, CudaPackage, ROCmPackage):
     """Tribol is an interface physics library."""
 
     homepage = "https://github.com/LLNL/Tribol"
     git      = "https://github.com/LLNL/Tribol.git"
 
-    version("develop", branch="develop", submodules=True)
+    version("develop", branch="develop", submodules=True, preferred=True)
```

Comment:
I don't think we would want to build with develop accidentally.

```suggestion
    version("develop", branch="develop", submodules=True)
```